engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1003,6 +1004,24 @@ - (UIMenu*)editMenuInteraction:(UIEditMenuInteraction*)interaction
                                    selector:@selector(captureTextFromCamera:)
                               suggestedMenu:suggestedMenu];
       }
+    } else if ([type isEqualToString:@"custom"]) {
+      NSString* customId = encodedItem[@"id"];
```

Comment:
Nit: Have you thought about naming this parameter callbackId just so it has the same name as it does in the framework? I'm not sure which name is better, just wanted to call it out.